### ISSUE TITLE
Buster come with new Rsyslog syntax − v6+

### ISSUE THREAD

**Issue by gardouille** [debops/debops#632]
The next Debian Stable − Buster, use a new Rsyslog's syntax (v6+), example with module load :
```
#
# Log messages sent to local UNIX socket
#
# provides support for local system logging
module(load="imuxsock")

#
# Log kernel messages
#
# provides kernel logging support
module(load="imklog" permitnonkernelfacility="on")

#
# Log periodic -- MARK -- messages
#
# provides --MARK-- message capability
#module(load="immark")
#module(load="immark" markmessageperiod="3600")
```

**Comment by drybjed** [comment]
The syntax seems to be supported on older Debian releases as well, so the role could be upgraded to it at some point.

**Comment by gardouille** [comment]
Extra info :
[Configuration Formats](https://www.rsyslog.com/doc/v8-stable/configuration/conf_formats.html)
[Converting older formats to advanced](https://www.rsyslog.com/doc/v8-stable/configuration/converting_to_new_format.html)

**Comment by drybjed** [comment]
@gardouille I'm currently finishing the rewrite of the role, I'll try to push it today or tommorow, as usual the documentation takes the longest...

**Comment by imrejonk** [comment]
Fixed in 09fb2ede01e8b7a7730e818dcfd706df705406c3
